### PR TITLE
18414 change usages of standard file stream and multi byte file stream to file  decorators

### PR DESCRIPTION
**Pull request by guillep**
Make the system use the new files:
 - remove usages of MultibyteFileStream, StandardFileStream and FileStream
 - pushing usage of Zn*Streams

On the way, I've fixed 64 bit tests... Decoupling them in two different issues is complicated...

### CODE REVIEW

**Comment by MarcusDenker** [comment]
it seems to fail on loading some font in the bootstrap:

[32] Finished atomic load
[32] Evaluated -> baseline [BaselineOfMorphicCore] >> postload:package:'Starting postload action'
[32] *** EXCEPTION ***
[32] FileDoesNotExistException: '/builds/workspace/and_branch_Pipeline_PR-1031-HRUS3GXYCZW7TEGUGDLPXU6FYO75CF26W3OGJI4LE4FRRYKQHCTQ/32/bootstrap-cache/resources/fonts/Bitmap DejaVu Sans 7 Regular.ston'

**Comment by guillep** [comment]
Yes I'm looking into it right now

File: src/Files-Prompt/MorphicUIManager.extension.st [hunk 1]
```diff
@@ -9,6 +9,7 @@ MorphicUIManager >> fileDoesNotExistUserHandling: filename [
 			('choose another name' translated).
 			('cancel' translated)}
 		title: filename, ' does not exist.'.
+	self halt.
```
Comment (dionisiydk) [inline src/Files-Prompt/MorphicUIManager.extension.st]: The halt in code

**Review by dionisiydk — COMMENTED** [review]
A lot of good changes. It can break something but it is important improvement

**Comment by dionisiydk** [comment]
In another issue https://pharo.fogbugz.com/f/cases/19026/Merge-FileDoesNotExist-and-FileDoesNotExistException we should clean file exception hierarchies. We have two now

**Pull request merged**